connector/src/main/java/org/jboss/as/connector/subsystems/resourceadapters/Constants.java

```diff
@@ -493,6 +493,7 @@ public void marshallAsElement(AttributeDefinition attribute, ModelNode resourceM
             .setRestartAllServices()
             .build();
 
+    @Deprecated
```

Comment:
This is package protected and unused so why not just delete it?

Reply:
fixed (deleted)